tests/integration/helpers.py

```diff
@@ -33,29 +33,31 @@ async def run_command_on_unit(unit, command: str) -> Optional[str]:
     return action.results.get("Stdout", None)
 
 
-def replica_set_client(replica_ips: List[str], password: str) -> MongoClient:
+def replica_set_client(replica_ips: List[str], password: str, app=APP_NAME) -> MongoClient:
     """Generates the replica set URI for multiple IP addresses.
 
     Args:
         replica_ips: list of ips hosting the replica set.
         password: password of database.
+        app: name of application which has the cluster.
```

Comment:
nit. What does `has the cluster` mean here?

Do you mean `cluster application name`? If so that might sound more obvious to a noob.

Reply:
sorry meant "hosts the the cluster"